requirements.txt

```diff
@@ -1,2 +1,30 @@
-flask
-peewee
+astroid==1.6.4
+Babel==2.5.3
+blinker==1.4
+click==6.7
+Flask==1.0.2
+Flask-Admin==1.5.1
+Flask-BabelEx==0.9.3
+Flask-Login==0.4.1
+Flask-Mail==0.9.1
+Flask-Principal==0.4.0
+Flask-Security==3.0.0
+Flask-SQLAlchemy==2.3.2
+Flask-WTF==0.14.2
+isort==4.3.4
+itsdangerous==0.24
+Jinja2==2.10
+lazy-object-proxy==1.3.1
+MarkupSafe==1.0
+mccabe==0.6.1
+passlib==1.7.1
+pbkdf2==1.3
+psycopg2-binary==2.7.4
+pylint==1.9.1
+pytz==2018.4
+six==1.11.0
+speaklater==1.3
+SQLAlchemy==1.2.7
+Werkzeug==0.14.1
+wrapt==1.10.11
+WTForms==2.1
```

Comment:
I don't think we need anything other than Flask, Flask-Security and Flask-SQLAlchemy. What are the others for?

Reply:
They are dependencies for things like Flask-Security or Flask-SQLAlchemy

Reply:
Alright. I forgot the whole dependency tree needs to be explicitly stated in `requirements.txt` with pip...